config/localTemplate.go

```diff
@@ -72,6 +72,7 @@ type Local struct {
 	BaseLoggerDebugLevel uint32 `version[0]:"1" version[1]:"4"`
 	// if this is 0, do not produce agreement.cadaver
 	CadaverSizeTarget uint64 `version[0]:"1073741824" version[24]:"0"`
+	CadaverDirectory  string `version[25]:""`
```

Comment:
need to add a new version to `Version` field. Also `go test ./config` will tell what else is missed

Reply:
My understanding is adding a new field with a new version should also define a new version otherwise it is inconsistent.

Reply:
`version[0]:""` would work, lets do it